site/en/blog/fledge-api/index.md

```diff
@@ -406,19 +445,30 @@ the following attributes can be changed:
 Any field not specified in the JSON will not be overwritten—only fields specified in the JSON get
 updated—whereas calling `navigator.joinAdInterestGroup()` overwrites any existing interest group.
 
-{% Aside "caution" %}
-
-The current implementation of FLEDGE in Chrome does not update interest groups automatically in the
-background.
+##### Manual updates
 
-For the time being, updates to interest groups owned by the current frame's origin can
-be triggered manually via `navigator.updateAdInterestGroups()`. Rate limiting prevents updates from
-happening too frequently: repeated calls to `navigator.updateAdInterestGroups()` don't do anything
-until the rate limit period (currently one day) has passed. The rate limit gets reset if
+Updates to interest groups owned by the current frame's origin can be triggered manually via
+`navigator.updateAdInterestGroups()`. Rate limiting prevents updates from happening too frequently:
+repeated calls to `navigator.updateAdInterestGroups()` don't do anything until the rate limit
+period (currently one day) has passed. The rate limit gets reset if
 `navigator.joinAdInterestGroup()` is called again for the same interest group `owner` and `name`.
 
+##### Automatic updates
+
+All interest groups loaded for an auction are updated automatically after an auction completes. For
+each owner with at least one interest group participating in an auction, it's as if
+`navigator.updateAdInterestGroups()` is called from an iframe whose origin matches that owner.
+
+Updates are best-effort, and can fail under the following conditions:
```

Comment:
I might move this so it's clear the best-effort properties apply equally to manually-triggered updates.

Reply:
Good call — I've moved this up.